test/e2e/flask/multichain-api/testHelpers.ts

```diff
@@ -30,6 +31,22 @@ export const DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS = {
       'build',
     ),
   ],
+  dappOptions: { numberOfDapps: 2 },
```

Comment:
do we have tests w/ 2 dapps yet?

Reply:
total nit but maybe this isn't a default fixture for multichain tests

Reply:
we have tests that connect to multiple tests, but only connect

Reply:
cleaned up here https://github.com/MetaMask/metamask-extension/pull/29464/commits/e1c7af07372a6d9ee40ce22856bcfc158d5cd874

Reply:
sorry I think you may have misunderstood. I'm wondering about multiple dapps not networks